### ISSUE TITLE
Dwarf Fortress v50.1x support

### ISSUE THREAD

**Issue by missionz3r0** [McArcady/lnp-forge#94]
DFv50.10 saw the release of a linux binary.
It looks like you are already updating the builder to support these latest versions and I would like to assist if possible.

So, to that end, do you even want the help? And if yes, where do you need it?

**Comment by McArcady** [comment]
Hey, thanks for proposing your help!
So here is the situation: the pack kind of builds and runs correctly with versions 50.10 and 50.11, but the problem is legal:  since 0.50, the new Linux version of DF seems to be legally protected against redistribution. So we may not have the right to release LNP packs that include the new DF Linux binary.

So for the moment I prefer not releasing new versions of the LinuxDwarfPack on Github or DFFD.

In the meantime you can always clone the repo, go to Actions > Build LinuxDwarfPack package > Run workflow.
It will build all the usual packages (AppImage, Debian, RPM... etc) in your github account.

Feel free to test the result and submit patches, we may update the scripts for our own use but probably not release the builds.

**Comment by missionz3r0** [comment]
Ah! That would be a problem. Fair for them to retain some rights after going so long giving it out for free.

@McArcady, what are your thoughts on releasing without the binary. As in, the user has to provide the tarball themselves? That should keep things in line with the new terms. 

---

For anyone else that stumbled over here with the same thoughts, looks like the new releases lack wording that was in the older release's readme.

Specifically:
```diff
Copyright (c) 2002-2021.  All rights are retained by Tarn Adams, save the following:
-you may redistribute the unmodified binary and accompanying files, provided you do so free of charge.
You may distribute modified text files from the data and raw folders (see the readme.txt in those
folders for more information). We'd appreciate it if you credit yourself or an alias somewhere for any
modifications to prevent confusion with vanilla DF (it helps with bug reports). 
- If you'd like to distribute a modified version of the game or portion of the archive and are worried
- about copyright infringement, please contact Tarn Adams at toadyone@bay12games.com.
```

**Comment by missionz3r0** [comment]
As an aside, I wonder why there is no mention of the change to the license on the forums?

As far as some searching has gotten me, I don't see it in any of the newb pack threads or elsewhere.

**Comment by ruizcrp** [comment]
Could it be that (amongst others) the graphics folder is empty when building with github actions? For example the LNP folder is quite empty.

**Comment by missionz3r0** [comment]
> Could it be that (amongst others) the graphics folder is empty when building with github actions? For example the LNP folder is quite empty.

I'm not sure what you mean? As discussed above, the license change is the problem.

**Comment by ruizcrp** [comment]
Yes, the license change is a problem for the redistribution. What I meant is - in the context of own-use plus having a newbie perspective without any prior experience linked to this repo - that the following said above does not work out of the box (without e.g. modifying the config-file and adding the missing elements):

> In the meantime you can always clone the repo, go to Actions > Build LinuxDwarfPack package > Run workflow.
